### ISSUE TITLE
Does it support special asscii charater?

### ISSUE THREAD

**Issue by rahuladream** [da03/Attention-OCR#107]
Can anyone explain the meaning of the following code

`ord(c) - 97 + 13 if ord(c) > 96 else ord(c) - 48 + 3`
where c is the any character

**Comment by khoon485** [comment]
Yes. It supports other ascii character as long as you fix some codes.
please refer to Attention OCR forked kspook's github.
